src/Datagrid/ProxyQuery.php

```diff
@@ -49,6 +49,13 @@ class ProxyQuery implements ProxyQueryInterface
      */
     protected $entityJoinAliases;
 
+    /**
+     * For BC reasons, this property is true by default.
```

Comment:
should we add something like "Remove on next major?

Reply:
Yes, but in a separate PR.